examples/favorites_advanced/getx_favorites_advanced/lib/feature_home/views/home_page.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:badges/badges.dart';
+import 'package:curved_navigation_bar/curved_navigation_bar.dart';
+
+import 'package:favorites_advanced_base/models.dart';
+import 'package:favorites_advanced_base/extensions.dart';
+
+import 'package:getx_favorites_advanced/base/ui_components/puppies_app_bar.dart';
+import 'package:getx_favorites_advanced/feature_home/controllers/navbar_controller.dart';
+import 'package:getx_favorites_advanced/feature_puppy/favorites/controllers/favorite_puppies_controller.dart';
+import 'package:getx_favorites_advanced/feature_puppy/favorites/views/favorites_page.dart';
+import 'package:getx_favorites_advanced/feature_puppy/search/views/search_page.dart';
+
+class HomePage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final navController = Get.find<NavbarController>();
+    return Scaffold(
+      appBar: PuppiesAppBar(),
+      body: AnimatedSwitcher(
+        duration: Duration(milliseconds: 300),
+        child: Obx(() =>
+        navController.selectedPage.type == NavigationItemType.search
+                  ? SearchPage()
+                  : FavoritesPage(),
+        ),
+      ),
+      bottomNavigationBar: CurvedNavigationBar(
+        color: Colors.blueAccent,
+        backgroundColor: Colors.transparent,
+        items: [
+          ...navController.items
+              .map((item) => Padding(
+                    padding: const EdgeInsets.all(8),
+                    child: item.asWidget(),
+                  ))
+              .toList()
+        ],
+        animationDuration: Duration(milliseconds: 300),
+        onTap: (index) => navController.selectPage(
+          index == 0 ? NavigationItemType.search : NavigationItemType.favorites,
+        ),
+      ),
+    );
+  }
+}
+
+extension NavigationItemToWitget on NavigationItem {
+  Widget asWidget() => type == NavigationItemType.favorites
+      ? Obx(
+          () => Get.find<FavoritePuppiesController>().count <= 0
+              ? type.asIcon()
+              : Badge(
+                  padding: const EdgeInsets.all(3),
+                  badgeContent: Text(
+                    Get.find<FavoritePuppiesController>().count.toString(),
+                    style: const TextStyle(
```

Comment:
@RomanovaPrime, please don't create an instance of `TextStyle` directly in the widget tree. The disadvantage of your approach is that there is no centralized place for all stylings. You can either use the `resources` library from the base package or you can create your own styles.dart in the project. Preferable all styles should reside in the `base` package so everyone can use the same styling.

Reply:
Added to resources in base packages